File: 242-valid-anagram/valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {

        if(s.size()!=t.size())
            return false;
        
        vector<int>countL(26);
        for(int i = 0 ; i < s.size() ; i++)
        {
            char ch = s[i];
            char ch1 = t[i];
            countL[ch-'a']++;
            countL[ch1-'a']--;
        }
        for(int i = 0 ; i < 26 ; i++)
        {
            if(countL[i]!=0)
                return false;
        }
        return true;
    }
};

File: 242-valid-anagram/valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        
        int[] arr = new int[26];
        
        for(char ch : s.toCharArray())
        {
            int index = (int)(ch) - (int)('a');
            arr[index]++;
        }
        for(char ch : t.toCharArray())
        {
            int index = (int)(ch) - (int)('a');
            arr[index]--;
        }
        for(int i = 0 ; i < 26 ; i++)
        {
            if(arr[i]!=0)
                return false;
        }
        return true;
    }
}